openwisp_controller/config/static/config/js/widget.js

```diff
@@ -153,6 +153,28 @@
         return error;
     };
 
+    var addMaxlengthClass = function() {
+        $('.jsoneditor input[maxlength]:not(.has-max-length)').addClass('has-max-length');
+    };
+    
+    var validateMaxLength = function() {
+        addMaxlengthClass();
+        var fields = $(".has-max-length");
+        fields.map((index, field) => {
+            if ($(field).is(":visible")) {
+                var maxLength = $(field).attr('maxLength');
+                $(field).off('input');
+                $(field).on('input', () => {
+                    if ($(field).val().indexOf('{{') > -1) {
+                        $(field).removeAttr('maxLength');
```

Comment:
Still, a problem remains i.e. If a user has config-variable as `WG0_subnet_ip_address_1` then copy-pasting won't work unless `{{` is present in the field value.
So, the user should have to write `{{` this first and then paste the variable.

Reply:
I see, I think we can live with that. @pandafy any idea if we can solve this at all?

Reply:
I provided a possible solution in the other comment, please check that.